main.py

```
@@ -74,6 +74,29 @@ def upgrade_packages(type):
+    """Create a merge request on GitLab"""
+    run_command(f'glab mr create --draft --title "{title}" --description "{description}"')
+
+def check_remote():
+    """Check the current remote"""
+    result = run_command("git remote -v", capture_output=True, suppress_output=False)
+    if "draft.beerntea.com" in result.stdout:
+        return "beerntea"
+    else:
+        return False
+    
+def create_merge_request(title, description):
```

Ruff (0.8.2)

Redefinition of unused `create_merge_request` from line 77

(F811)